plugins/main/windows/overlay/overlay_windows.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2014 CNI authors
```

Comment:
This file is almost identical to the l2_bridge_windows.go file, can we try to share some copy rather than duplicating so much?